experimental/addons/README.md

```diff
@@ -1,19 +1,32 @@
 # KubeVela experimental addons
 
-This dir is KubeVela official experimental addon-registry which will contain hasn't verified stable KubeVela addon.
+This dir is KubeVela official experimental addon-registry which will contain some KubeVela addons which haven't verified stable.
 
-Addon files in this dir files will be synced to alibaba-oss but will not be set in KubeVela by default.
+Addon files in this dir files will be synced to alibaba-oss. Addons in this  dir will not be set in KubeVela by default.
 
-You can add this addon registry by vela cli then use these addons.
+You can add this addon registry by vela cli and use them.
 
 ```shell
-$ vela addon registry add experimental --type=git --gitUrl=https://github.com/oam-dev/catalog/ --path=experimental/addons
+$ vela addon registry add experimental --type oss --ossEndpoint=https://addons.kubevela.net --path=experimental/
```

Comment:
Can we omit parse the `path` automatically？ And we already decalre it as OSS type, then we shouldn't redundant the parameter as `ossEndpoint`

The Command should be something like:

```
vela addon registry add experimental --type oss --endpoint=https://addons.kubevela.net/experimental
```

Reply:
Actually This is not a valid path: https://addons.kubevela.net/experimental, path=experimental is used like `https://addons.kubevela.net?prefix=experimental/`. So I don't argree to let user to fill in this URL.